gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/CodeableConceptMapperTest.java

```diff
@@ -395,7 +397,7 @@ public void mapNoSnomedCodeWithOriginalText() {
         CodeableConcept codeableConcept = codeableConceptMapper.mapToCodeableConcept(codedData);
 
         assertThat(codeableConcept.getText()).isEqualTo("Inactive Problem, minor");
-        assertThat(codeableConcept.hasCoding()).isFalse();
+        assertThat(codeableConcept.getCoding()).hasSize(EXPECTED_CODING_SIZE);
```

Comment:
I _hate_ with a passion the magic numbers code style thing. I feel like this test is more confusing now.